benchmarks/python/test_matmul.py

```diff
@@ -38,26 +37,20 @@ def test_matmul_nvf_benchmark(
 ):
     m, n, k, layout = config
 
-    clear_cuda_cache()
+    a = torch.randn(m, k, device="cuda", dtype=dtype)
```

Comment:
Will the changes in the this file be superseded by the other PR?

Reply:
I will merge it manually

Reply:
Merged